packer.json

```diff
@@ -11,14 +13,11 @@
 		}
 	],
 	"provisioners": [
-        {
-            "type": "file",
-            "source": "jdk",
-            "destination": "/tmp"
-        },
 		{
 			"type": "shell",
-			"script": "scripts/install_java.sh"
+			"script": "scripts/install_java.sh",
+      "environment_vars": ["PROXY_UNAME={{user `proxy_uname`}}",
```

Comment:
Should we get this from the environment of the host rather than relying on the person to type it on the command line when running Packer?  (This would allow a lot of time savings from going and looking this information up each time.)